README.md

```diff
@@ -1 +1,140 @@
 # cli-options-parser
+
+[![Software License](https://img.shields.io/badge/license-BSD--3-brightgreen.svg?style=flat-square)][:license:]
+
+## What's Phalcon?
+
+Phalcon PHP is a web framework delivered as a C extension providing high performance and lower resource consumption.
```

Comment:
What's Phalcon? It is command line arguments parser which uses Phalcon. This project does not provide Phalcon. This section should be removed.